boards/adi/apard32690/apard32690_max32690_m4.dts

```diff
@@ -0,0 +1,160 @@
+/*
+ * Copyright (c) 2024 Analog Devices, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+
+#include <adi/max32/max32690.dtsi>
+#include <adi/max32/max32690-pinctrl.dtsi>
+#include <zephyr/dt-bindings/gpio/adi-max32-gpio.h>
+#include <zephyr/dt-bindings/input/input-event-codes.h>
+
+/ {
+	model = "Analog Devices AD-APARD32690-SL";
+	compatible = "adi,apard32690";
+
+	chosen {
+		zephyr,console = &uart0;
+		zephyr,shell-uart = &uart0;
+		zephyr,bt-uart = &uart3;
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		blue_led: blue_led {
+			gpios = <&gpio2 1 GPIO_ACTIVE_HIGH>;
+			label = "LED1";
+		};
+		red_led: red_led {
+			gpios = <&gpio0 12 GPIO_ACTIVE_HIGH>;
+			label = "LED2";
+		};
+		green_led: green_led {
+			gpios = <&gpio0 11 GPIO_ACTIVE_HIGH>;
+			label = "LED3";
+		};
+	};
+
+	buttons {
+		compatible = "gpio-keys";
+		usr_btn: usr_btn {
+			gpios = <&gpio1 27 (GPIO_PULL_DOWN | GPIO_ACTIVE_HIGH)>;
+			label = "S2";
+			zephyr,code = <INPUT_KEY_0>;
+		};
+	};
+
+	aliases {
+		led0 = &blue_led;
+		led1 = &red_led;
+		led2 = &green_led;
+		sw0 = &usr_btn;
+	};
+};
+
+&clk_ipo {
+	status = "okay";
+};
+
+&clk_ibro {
+	status = "okay";
+};
+
+&gpio0 {
+	status = "okay";
+};
+
+&gpio1 {
+	status = "okay";
+};
+
+&gpio2 {
+	status = "okay";
+};
+
+&gpio3 {
+	status = "okay";
+};
+
+&gpio4 {
+	status = "okay";
+};
+
+&uart0 {
+	clock-source = <ADI_MAX32_PRPH_CLK_SRC_IBRO>;
+	pinctrl-0 = <&uart0a_tx_p2_12 &uart0a_rx_p2_11>;
+	pinctrl-names = "default";
+	current-speed = <115200>;
+	data-bits = <8>;
+	parity = "none";
+	status = "okay";
+};
+
+&uart3 {
+	status = "okay";
+	current-speed = <1000000>;
+	data-bits = <8>;
+};
```

Comment:
No pinctrl for this node? I also see that this node is chosen for `zephyr,bt-uart`. Have you tested this?

Reply:
I have not tested the Bluetooth functionality. The `zephyr,bt-uart` property was only added since the CI  would fail to compile `samples/bluetooth/peripheral_hr` otherwise. Is there a better fix for this?

Reply:
`zephyr,bt-uart` gets set in `boards/shields/frdm_kw41z/frdm_kw41z.overlay` so you shouldn't need to set it here. You should be able to fix the CI failure by cherry-picking #74038. You'll then also be able to remove the `uart3` node which I presume you're using as a placeholder just to fake out CI.